bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -124,8 +124,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <organic_frac_squared phys="clm4_5" >.true.</organic_frac_squared>
 <organic_frac_squared phys="clm5_0" >.false.</organic_frac_squared>
 
-<soil_layerstruct phys="clm5_0">20SL_8.5m</soil_layerstruct>
-<soil_layerstruct phys="clm4_5">10SL_3.5m</soil_layerstruct>
+<soil_layerstruct structure="fast">5SL_3m</soil_layerstruct>
+<soil_layerstruct structure="standard" phys="clm5_0">20SL_8.5m</soil_layerstruct>
+<soil_layerstruct structure="standard" phys="clm4_5">10SL_3.5m</soil_layerstruct>
```

Comment:
This is where the CLM_PHYSICS version and the structure are conflated into one thing. And that could be OK. But, another way to do this would be that 10SL_3.5m is structure version standard4.5, and 20SL_8.5m is standard5.0. The compset would pick which structure version goes with which physics. But, then you can refer to combinations like -- "I'm running clm5.0 physics with standard4.5 structure". Rather than having to say -- "I'm running clm5.0 physics with standard structure and the 10SL_3.5m soil depth that was in clm4.5 physics". For previous structure, you might as well use the physics numbering that was used in the past, but it wouldn't have to align that way for moving forward -- unless that is useful.

Reply:
OK, from my discussions with people conflating physics version and structure into one thing is OK, because default structure isn't going to change very often.